src/components/controls/vector-styles/PropertySelect_New.svelte

```diff
@@ -0,0 +1,135 @@
+<script lang="ts">
```

Comment:
Can you set more approriate name for `PropertySelect_New.svelte`? I think `_2` or `_New` are not a good name. It may confuse people

For example, `PropertySelectFilter.svelte`